string target_node_id
---
bool success
string message
---
